core/modules/widgets/range.js

```diff
@@ -111,15 +202,17 @@ Selectively refreshes the widget if needed. Returns true if the widget or any of
 */
 RangeWidget.prototype.refresh = function(changedTiddlers) {
 	var changedAttributes = this.computeAttributes();
-	if(changedAttributes.tiddler || changedAttributes.field || changedAttributes.index || changedAttributes['min'] || changedAttributes['max'] || changedAttributes['increment'] || changedAttributes["default"] || changedAttributes["class"] || changedAttributes.disabled) {
+	if(changedAttributes.tiddler || changedAttributes.field || changedAttributes.index || changedAttributes['min'] || changedAttributes['max'] ||
```

Comment:
@pmario if we need to call` refreshSelf()` if any attributes have changed, it is easier to just check the count:

```
if($tw.utils.count(changedAttributes) > 0) {
    this.refreshSelf();
    return true;
}
```

Reply:
Isn't this the case for every widget?

Reply:
@pmario We just can't use this trick when we only want to call `refreshSelf()` on some attributes changing but not for others. So there is probably room for some refactoring and optimization in other core widgets.